x/staking/testutil/helpers.go

```diff
@@ -121,7 +122,7 @@ func (sh *Helper) CheckValidator(addr sdk.ValAddress, status stakingtypes.BondSt
 
 // CheckDelegator asserts that a delegator exists
 func (sh *Helper) CheckDelegator(delegator sdk.AccAddress, val sdk.ValAddress, found bool) {
-	_, ok := sh.k.GetDelegation(sh.Ctx, delegator, val)
+	_, ok := sh.k.Delegations.Get(sh.Ctx, collections.Join(delegator, val))
```

Comment:
This does not return a bool anymore so this is incorrect.

Reply:
Yeah, BTW this helper method is not being used anywhere, just removed it.